cli/src/infragpt/llm/providers/anthropic_provider.py

```
@@ -0,0 +1,211 @@
+
+class AnthropicProvider(BaseLLMProvider):
+    """Anthropic provider using official Python SDK."""
+    
+    def _initialize_client(self, **kwargs):
+        """Initialize Anthropic client."""
+        return Anthropic(api_key=self.api_key)
+    
+    def validate_api_key(self) -> bool:
+        """Validate API key with a simple test call."""
+        try:
+            response = self._client.messages.create(
```

Ruff (0.12.2)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)